### ISSUE TITLE
Typo in Readme?

### ISSUE THREAD

**Issue by mbrochh-win** [timmh/react-native-camera#6]
The README suggests to use this line:

```
project(':include ':com.lwansbrough.ReactCamera').projectDir = new File(settingsDir, '../node_modules/react-native-camera/android')
```

However, that is giving me errors, so I changed it into this (as suggested in a thread in the original repo's issue tracker)

```
project(':com.lwansbrough.ReactCamera').projectDir = new File(settingsDir, '../node_modules/react-native-camera/android')
```

Should the readme be updated?

**Comment by guillaumemorin** [comment]
:+1: Agree. First line give me an error too. Second one is the good one!

**Comment by timmh** [comment]
Right, I have no idea how that line made it in there...
Anyway, thanks for reporting this issue!
Fixed with 2d56ab0
